colorspace/color.py

```diff
@@ -71,7 +75,8 @@ def cluster(fragments):
     coloring = color(nodes)
 
     # Arrange the fragments w/in groups
-    sorted_nodes = sorted(nodes, key=lambda node: node.angle)
+    # TODO: Is this dead code? Should it be coloring instead of nodes?
+    sorted_nodes = sorted(nodes, key=lambda node: node.angle) # type: ignore
```

Comment:
Coloring is mutative (probably shouldn't be?), so this still works. The sort's needed for the following "induce order" logic. It's semi-dead insofar as I think cluster-level colors will be more meaningful and useful than the more granular fragment-level colors.